src/ru/skypro/Main.java

```diff
@@ -3,6 +3,26 @@
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+//   №1
+        int j = 0;
+        while(j < 10){
+            j++;
+            System.out.println(j);
+        }
+        System.out.println(" ");
+        for(int i = 10; 1 <= i;i-- ){
```

Comment:
обычно в условии пишут сначала переменную, затем константу
```suggestion
        for(int i = 10; i >= 1; i-- ){
```